htsget-http-actix/src/config.rs

```diff
@@ -22,6 +30,10 @@ pub struct Config {
   pub htsget_ip: String,
   #[serde(default = "default_path")]
   pub htsget_path: PathBuf,
+  #[serde(default = "default_regex")]
+  pub htsget_regex: String,
+  #[serde(default = "default_replacement")]
```

Comment:
You mean default substitution? IIRC regexes for `sed` have `'s'` for "substitution", minor detail.

Reply:
I think I have also seen replacement, but yeah, substitution is more common